src/components/UI/FilterBar/FilterBar.css

```diff
@@ -0,0 +1,22 @@
+.FilterBar {
+  display: flex;
+  margin-top: 16px;
+  border-radius: 4px;
+  /* TODO: remove importants */
```

Comment:
Did you intend to leave these TODOs?

Reply:
Nope, but I didn't want to stall the feature.
Depends on when we need this, I can fix it probably on Friday.